config/database.yml

```diff
@@ -2,27 +2,3 @@ default: &default
   adapter: postgresql
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-
-development:
```

Comment:
Deleting the development and test environment requires us to set `DATABASE_URL` to be able to run the application or tests locally.